rtloader/common/builtins/_util.c

```diff
@@ -103,18 +103,21 @@ static void raiseEmptyOutputError()
     empty an exception will be raised: the error will be set in the interpreter and NULL will be
     returned.
 */
-PyObject *subprocess_output(PyObject *self, PyObject *args)
+PyObject *subprocess_output(PyObject *self, PyObject *args, PyObject *kw)
```

Comment:
The doxygen documentation above has to be updated to account for `kwargs`.

Reply:
Indeed !
Done: https://github.com/DataDog/datadog-agent/pull/5987/commits/fedca6c193ba21449f8046b0bf5dcdbd616e53b6.